src/Bindings/PluginManager.cpp

```diff
@@ -788,11 +788,11 @@ bool cPluginManager::CallHookPlayerFished(cPlayer & a_Player, const cItems & a_R
 
 
 
-bool cPluginManager::CallHookPlayerFishing(cPlayer & a_Player, cItems a_Reward)
+bool cPluginManager::CallHookPlayerFishing(cPlayer & a_Player, cItems a_Reward, int ExperienceAmount)
```

Comment:
I'm a bit confused of why a_Reward here is not a (const) reference. Is this because in FISHED the reward is already settled so we use const reference for performance. And in FISHING we use a copy instead becuase the rewards will change while fishing so sending back and forth references isn't ideal? I might be completely wrong about this but I'm curious :p. 
And what about the ExperienceAmount in this case.

Reply:
You've found a bug :P This should be a non-const reference as well.

Reply:
Hahaha okay didn't think about that.